web/src/beta/features/Editor/Publish/PublishToolsPanel/PublishModal/index.tsx

```diff
@@ -242,13 +258,14 @@ export default PublishModal;
 const Section = styled.div<{ disabled?: boolean }>`
   display: flex;
   flex-direction: column;
+  padding: 16px;
   gap: 8px;
   margin-bottom: ${`${spacingSizes["normal"]}px`};
   opacity: ${({ disabled }) => disabled && "0.6"};
   cursor: ${({ disabled }) => disabled && "not-allowed"};
 `;
 
-const Subtitle = styled(Text)`
+const Subtitle = styled(Typography)`
```

Comment:
we can not override components style. define wrapper and wrap the component if you want to add some styles 
```suggestion
const SubtitleWrapper = styled("div")`
```